common/general/general_test.go

```diff
@@ -464,3 +467,13 @@ func TestCheckExpiredEncryptionCertValidityDuringContractEncryption(t *testing.T
 	_, err = CheckEncryptionCertValidityForContractEncryption(string(data))
 	assert.Contains(t, err.Error(), "Encryption certificate has already expired")
 }
+
+// Testcase to check GzipInitData() is able to gzip data
+func TestGzipInitData(t *testing.T) {
+	data, err := GzipInitData(sampleStringToGzip)
+	if err != nil {
+		t.Errorf("failed to create gzipped data - %v", err)
+	}
+	assert.NotEmpty(t, data)
```

Comment:
Can we provide an independently generated gzipped version (in the samples directory as a file) to properly test this rather than just checking that it's not empty?